### DISCUSSION TITLE
ElasticSearch cluster with multiple nodes

### DISCUSSION THREAD

**Discussion by nagyobbaron** [sherifabdlnaby/elastdocker#57] [Questions & Troubleshooting]
Hello, 
I'm trying to figure out how to create multiple elasticsearch instances to form an elasticsearch cluster.
Can you please give me a hint? 
Thank you.

**Comment by sherifabdlnaby** [comment]
`make nodes` for a starter.
And check `docker-compose.nodes.yml`

**Comment by sherifabdlnaby** [comment]
This implementation is just for experimentation though.
